frontend/storybook/src/components/entry/imagelabel/one.ts

```diff
@@ -0,0 +1,38 @@
+import "@elements/entry/admin/images/meta/page";
+import { LeftLabel } from "~/components/entry/imagelabel/admin/images/image-label-left";
+import {LabelRight} from "~/components/entry/imagelabel/admin/images/image-label-right";
+import "@elements/core/titles/variants/underlined-title";
+import { Rectangle } from "~/components/core/buttons/rectangle";
+
+export default {
+  title: 'Full Pages/Image Label',
+}
+
+interface Args {
+}
+
+const DEFAULT_ARGS:Args = {
+}
+
+
+const STR_NEXT = "Next";
```

Comment:
Do these ever change at runtime? If not - should be in an element

Reply:
This is the text for the button that does change in runtime I believe. Added from existing element.

Reply:
OK - fine for now, will be refactored later with https://github.com/ji-devs/ji-cloud/issues/769